packetbeat/docs/packetbeat-geoip.asciidoc

```diff
@@ -0,0 +1,165 @@
+[[packetbeat-geoip]]
+== Exporting GeoIP Information 
+
+You can use Packetbeat along with the
+{plugindoc}/ingest-geoip.html[ingest geoIP processor plugin] in Elasticsearch 
+to export geographic location information about source IPs for incoming HTTP
+requests. Then you can use this info to visualize the location of your
+clients on a map in Kibana.
+
+Prior to version 5.0, Packetbeat provided a `geoip` configuration option for
+exporting geoIP information about the source IPs.  Starting with 5.0, the
+`geoip` configuration option in Beats is deprecated in favor of using the
+ingest geoIP processor plugin. This plugin adds information about the
+geographical location of IP addresses, based on data from the Maxmind GeoLite2
+City Database. Using this plugin provides you with greater flexibility and
+control over the information that's returned by the geoIP lookup. 
+
+//REVIEWERS: trying to put some words around why the plugin is better, but I'm not sure if this is completely true. ^^
```

Comment:
The benefit is that you don't need to install the geoip database on all the Beats, and you can install it only on your Elasticsearch.